cmd/api/src/api/v2/database_wipe.go

```diff
@@ -0,0 +1,243 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package v2
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gofrs/uuid"
+	"github.com/specterops/bloodhound/dawgs/graph"
+	"github.com/specterops/bloodhound/dawgs/ops"
+	"github.com/specterops/bloodhound/log"
+	"github.com/specterops/bloodhound/src/api"
+	"github.com/specterops/bloodhound/src/model"
+)
+
+type DatabaseWipe struct {
+	DeleteCollectedGraphData bool `json:"deleteCollectedGraphData"`
+	DeleteHighValueSelectors bool `json:"deleteHighValueSelectors"`
+	DeleteFileIngestHistory  bool `json:"deleteFileIngestHistory"`
+	DeleteDataQualityHistory bool `json:"deleteDataQualityHistory"`
+	AssetGroupId             int  `json:"assetGroupId"`
```

Comment:
If the purpose of this endpoint is to "delete high value selectors" - why is the onus on the requestor to provide the correct asset group ID? Can the service not derive the asset group ID for the appropriate asset group?

Reply:
I forgot to consider in the AC for this ticket that multiple groups exist (and will become more prevalent in the future).

Would it make more sense to leave it this way, change `DeleteHighValueSelectors` to `DeleteGroupSelectors` and then move `AssetGroupId` to an array/list for improved long-term supportability (with an 'all' option)? I recognize this would be a change to AC and scope, and I am happy to discuss it.

Reply:
@StephenHinck it is your feature/call to make. I think that what @brandonshearin has implemented satisfies your original AC but if that is no longer desired then we will need time to discuss and make adjustments. Consider that once this changeset is in and released it will be harder to make changes to the API contract.